libs/sdk/RNRudderSdk.podspec

```diff
@@ -19,7 +19,7 @@ Pod::Spec.new do |s|
   s.requires_arc = true
 
   s.dependency "React"
-  s.dependency "Rudder", "~> 1.13"
```

Comment:
I thought to keep it as `'~> 1.19`, as it will be compatible with other possible fixes that we are going to release w.r.t. encrypt database task. (I'm referring to the recent SQLITE issue, if we are going to provide the fix.) I also thought to mention the version info directly into the doc.

If we are not going to provide any further fix related to this feature, I'll update it, as suggested, `>= 1.19.2`.